File: livingrimoire start here/LivinGrimoire java/AXJava/ElizaDeducerInitializer.java
package AXJava;

import java.util.ArrayList;
import java.util.Collections;

public class ElizaDeducerInitializer extends ElizaDeducer{
    public ElizaDeducerInitializer() {
        ref.put("am", "are");
        ref.put("was", "were");
        ref.put("i", "you");
        ref.put("i'd", "you would");
        ref.put("i've", "you have");
        ref.put("my", "your");
        ref.put("are", "am");
        ref.put("you've", "Ihave");
        ref.put("you'll", "I will");
        ref.put("your", "my");
        ref.put("yours", "mine");
        ref.put("you", "i");
        ref.put("me", "you");
        reflections = Collections.unmodifiableMap(ref);
        ArrayList<PhraseMatcher> babbleTmp = new ArrayList<PhraseMatcher>();
        ArrayList<AXKeyValuePair> kvs = new ArrayList<>();
        kvs.add(new AXKeyValuePair("what is a {0}","{0} is a {1}"));
        kvs.add(new AXKeyValuePair("explain {0}","{0} is a {1}"));
        babbleTmp.add(new PhraseMatcher("(.*) is (.*)", kvs));
        babble2 = Collections.unmodifiableList(babbleTmp);
    }
}

File: livingrimoire start here/LivinGrimoire Kotlin/kotlin/auxiliary_modules/ElizaDeducerInitializer.kt
package auxiliary_modules

import java.util.*

class ElizaDeducerInitializer : ElizaDeducer() {
    init {
        val babbleTmp = ArrayList<PhraseMatcher>()
        val kvs = ArrayList<AXKeyValuePair>()
        kvs.add(AXKeyValuePair("what is {0}", "{0} is {1}"))
        kvs.add(AXKeyValuePair("explain {0}", "{0} is {1}"))
        babbleTmp.add(PhraseMatcher("(.*) is (.*)", kvs))
        val or1 = ArrayList<AXKeyValuePair>() // simple or
        or1.add(AXKeyValuePair("{0}", "{2}"))
        or1.add(AXKeyValuePair("{1}", "{2}"))
        babbleTmp.add(PhraseMatcher("if (.*) or (.*) than (.*)", or1)) // end or1
        val if1 = ArrayList<AXKeyValuePair>() // simple if
        if1.add(AXKeyValuePair("{0}", "{1}"))
        babbleTmp.add(PhraseMatcher("if (.*) and (.*) than (.*)", if1)) // end if1
        val because1 = ArrayList<AXKeyValuePair>() // simple because
        because1.add(AXKeyValuePair("{1}", "i guess {0}"))
        babbleTmp.add(PhraseMatcher("(.*) because (.*)", because1)) // end because
        babble2 = Collections.unmodifiableList(babbleTmp)
    }
}